libs/ddd-angular/src/generators/feature/schema.json

```diff
@@ -16,13 +16,12 @@
     },
     "domain": {
       "type": "string",
-      "description": "",
-      "$default": {
-        "$source": "argv",
-        "index": 0
-      },
+      "description": "The domain to add the feature",
       "x-prompt": "What domain would you like to use?"
     }
   },
-  "required": ["name", "domain"]
-}
+  "required": [
```

Comment:
Don't features also need an app specified?

Reply:
NOT ANYMORE!!!